src/sentry/static/sentry/app/views/alerts/details/chart.tsx

```diff
@@ -75,7 +113,7 @@ export default class Chart extends React.PureComponent<Props> {
           {
             // e.g. Events or Users
             seriesName,
-            dataArray: chartData?.slice(0, -1),
+            dataArray: chartData,
```

Comment:
Why remove this?

Reply:
I'm adding the last point back in. This also "resolves": https://app.asana.com/0/1143846759074121/1169079974489955